package.json

```diff
@@ -36,7 +36,7 @@
   },
   "homepage": "https://github.com/axiomhq/next-axiom#readme",
   "peerDependencies": {
-    "next": "^12.1.4"
+    "next": "^12.2.0"
```

Comment:
Yeah so let's keep the top-level version at ^12.1.4. 

> This confuses me regarding the nested middleware error, on the 12.1.4 it could use pages/_middleware, right? but not the /middleware.js?
Yes

Reply:
done